examples/decode_exif_metadata.cc

```diff
@@ -72,7 +73,7 @@ bool DecodeJpegXlExif(const uint8_t* jxl, size_t size,
       exif->resize(exif->size() + kChunkSize);
       JxlDecoderSetBoxBuffer(dec.get(), exif->data() + output_pos,
                              exif->size() - output_pos);
-    } else if (status == JXL_DEC_SUCCESS) {
+    } else if (status == JXL_DEC_BOX_COMPLETE) {
```

Comment:
Doesn't this also need a change to the Subscribe call?

Reply:
It does, and there is.